File: FlightsApp/Backend/Flights/DTOs/BuyTicketDTO.cs
namespace Flights.DTOs
{
    public class BuyTicketDTO
    {
        public string userId { get; set; }
        public string flightId { get; set; }
        public int numberOfTickets { get; set; }
        public float price { get; set; }

        public BuyTicketDTO()
        {
        }

        public BuyTicketDTO(string userId, string flightId, int numberOfTickets,float price)
        {
            this.userId = userId;
            this.flightId = flightId;
            this.numberOfTickets = numberOfTickets;
            this.price = price;
        }
    }
}

File: FlightsApp/Frontend/Flights/src/app/model/buyTicketDto.ts
export class BuyTicketDto{
  userId?: string
  flightId?: string
  numberOfTickets?: string
  price?: number
  public constructor(obj?:any){
    if(obj){
      this.userId = obj.userId
      this.flightId = obj.flightId
      this.numberOfTickets = obj.numberOfTickets
      this.price = obj.price
    }
  }
}